static/src/components/setting/task_setting.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div class="stats-config row">
+    <div class="col-md-12">
+      <div class="form-group row">
+        <div class="col-md-6">
+          <p> task scheduler </p>
+        </div>
+        <div class="col-md-6 pull-right">
+          <select
+            v-model="sched"
+            class="form-control input-sm"
+          >
+            <option
+              v-for="opt in getOptions()"
+              :key="opt.text"
+              :value="opt.value"
+            >
+              {{ opt.text }}
+            </option>
+          </select>
+        </div>
+      </div>
+      <div class="form-group row">
+        <div class="col-md-6">
+          <p> randomize order with same priority </p>
+        </div>
+        <div class="col-md-6 pull-right">
+          <label class="switch">
```

Comment:
This is relying on css defined on a different component. Perhaps it is convenient to move the switch style to a scss file so it can be reused across different pages.

Reply:
Curious, why you decided to use switches instead of checkboxes?

Reply:
Agree, already move the style to a sass file. and it seems that "switch" is the official name for this UI component.